components/cd-bleed/cd-bleed.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Update the --gho-bleed-scrollbar-width css variable when the <html> element
+ * is resized, including when the scrollbar is initially added.
+ */
+(function () {
```

Comment:
I prefer to name all functions, since it allows stack traces to better identify where errors arise. I typically call the top-level IIFE `iife()` — it doesn't matter if the name is reused in multiple files since it's wrapped in parentheses.